content/en/docs/reference/glossary/api-group.md

```diff
@@ -0,0 +1,18 @@
+---
+title: API Group
+id: api-group
+date: 2019-09-02
+full_link: /docs/concepts/overview/kubernetes-api/#api-groups
+short_description: >
+In the Kubernetes API, an API group is a set of related API paths. You can enable or disable each API group by changing the configuration of your API server. You can also disable or enable paths to specific resources.
```

Comment:
- this is quite long for a short description
- the line needs indenting so that this file is valid YAML (it's actually a YAML file that contains Markdown)

Reply:
@sftim I have updated short description. Let me know what you think.